src/runtime_src/core/tools/common/ReportCu.cpp

```diff
@@ -70,9 +74,28 @@ get_cu_status(uint32_t cu_status)
   return pt;
 }
 
+static void
+schedulerUpdateStat(xrt_core::device *device)
+{
+  try {
+    // lock xclbin
+    auto uuid = xrt::uuid(xrt_core::device_query<xrt_core::query::xclbin_uuid>(device));
+    device->open_context(uuid.get(), static_cast<unsigned int>(-1), true);

```

Comment:
Please don't use -1 to initialize an unsigned int to all 1s.   Instead please use:
`std::numeric_limits<unsigned int>::max()`

Reply:
fixed